src/udev/udevadm-trigger.c

```diff
@@ -226,6 +240,8 @@ static int help(void) {
                "  -y --sysname-match=NAME           Trigger devices with this /sys path\n"
                "     --name-match=NAME              Trigger devices with this /dev name\n"
                "  -b --parent-match=NAME            Trigger devices with that parent device\n"
+               "  -d --database-match               Trigger devices having a udev db entry\n"
+               "  -x --database-nomatch             Trigger devices not having a udev db entry\n"
```

Comment:
this doesn't look like the most common switch to use, hence please drop the short switch, leave only the long one.

We typically only assign short switches to stuff we know or expect to be used frequently.